src/lib/fileutils.ts

```diff
@@ -231,10 +261,25 @@ export const starterAzurePipelines = async (opts: {
                     `az extension add --name azure-devops`,
                     ``,
                     `echo 'az repos pr create --description "Updating $PROJECT_NAME_LOWER to $(Build.SourceBranchName)-$(Build.BuildNumber)."'`,
-                    `az repos pr create --description "Updating $PROJECT_NAME_LOWER to $(Build.SourceBranchName)-$(Build.BuildNumber)."`
+                    `az repos pr create --description "Updating $PROJECT_NAME_LOWER to $(Build.SourceBranchName)-$(Build.BuildNumber)."`,
+                    ``,
+                    `# Update introspection storage with this information, if applicable`,
+                    `if [ -z "$(ACCOUNT_NAME)" -o -z "$(ACCOUNT_KEY)" -o -z "$(TABLE_NAME)" -o -z "$(PARTITION_KEY)" ]; then`,
+                    `echo "Introspection variables are not defined. Skipping..."`,
+                    `else`,
+                    `latest_commit=$(git rev-parse --short HEAD)`,
+                    `tag_name="$BUILD_REPO_NAME:$(Build.SourceBranchName)-$(Build.BuildNumber)"`,
+                    `echo "Downloading SPK"`,
+                    `curl https://raw.githubusercontent.com/Microsoft/bedrock/master/gitops/azure-devops/build.sh > build.sh`,
+                    `chmod +x build.sh`,
+                    `. ./build.sh --source-only`,
+                    `get_spk_version`,
+                    `download_spk`,
+                    `./spk/spk deployment create  -n $(ACCOUNT_NAME) -k $(ACCOUNT_KEY) -t $(TABLE_NAME) -p $(PARTITION_KEY) --p2 $(Build.BuildId) --hld-commit-id $latest_commit --env $BRANCH_NAME --image-tag $tag_name`,
+                    `fi`
                   ]),
                   displayName:
-                    "Download Fabrikate, Update HLD, Push changes, Open PR",
+                    "Download Fabrikate, Update HLD, Push changes, Open PR, optionally push to Spektate storage",
```

Comment:
Should we mention this in the introspection documentation? I'm thinking it can go in the getting started with introspection guide.